index.md

```diff
@@ -32,7 +32,7 @@ consumers of your library:
   with [`MINOR`](http://semver.org/#summary) in semantic versioning).
 3. **BREAKING CHANGE:** a commit that has the text `BREAKING CHANGE:` at the beginning of its optional body or footer section introduces a breaking API change (correlating with [`Major`](http://semver.org/#summary) in semantic versioning). A breaking change can be
   part of commits of any _type_. e.g., a `fix:`, `feat:` & `chore:` types would all be valid, in addition to any other _type_.
-4. Others: commit _types_ other than `fix:` and `feat:` are allowed, for example [the Angular convention](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit) recommends `docs:`, `style:`, `refactor:`, `perf:`, `test:`, `chore:`, but these tags are not mandated by the conventional commits specification.
+4. Others: commit _types_ other than `fix:` and `feat:` are allowed, for example [the Angular convention](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit) recommends `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others, but these tags are not mandated by the conventional commits specification, and have no effect in semantic versioning.
```

Comment:
This is not true, a refactor may introduce BC and effect the semver

Reply:
I think refactor *can* introduce BREAKING CHANGE. But only internal. For software that relies on the private API it could break.

Reply:
In fact, you could introduce BREAKING CHANGE with any type. So this should be:

```
and have no effect in semantic versioning if there's no BREAKING CHANGE.
```

Reply:
Yup, @stevemao agree with your suggestion!